docs/irdl.ipynb

```diff
@@ -22,6 +22,46 @@
   {
    "cell_type": "code",
    "execution_count": 1,
+   "id": "1c670f99",
```

Comment:
This is sth we definitely do not want

Reply:
What don't we want and why?

Reply:
I think he means that we want to remove the output of a cell.
We prefer cleaning the outputs first before pushing it!

Reply:
The output should not be there, an din my opinion, neither the installation.
It doesn't look like a good practice.
Have you seen any other projects doing sth similar?

Reply:
Got it, I'll clean the outputs.
I have little experience in notebooks so did not see this yet, but I know some do in JupyterLite context, as it is a limitation of its Python kernel Pyodide.
Pyodide _has to_ install packages at runtime, and for every restart. Currently the only way around that is to rebuild a whole Pyodide kernel with our package included, which is not yet easily automated. They are thinking about more straightforward ways, so I was thinking about waiting for those.

There may be some way to hide the pip install command from the user, but still run it at kernel startup time. Would that option satisfy you?